sass/index.scss

```diff
@@ -1,166 +1,632 @@
+// Variables
+// Colors
+$dark-blue: #193787;
+$blue: #1447C8;
+$light-blue: #44C6E6;
+$off-white: #F6F8FA;
+$black: #081646;
+$orange: #FF9C4A;
 
-// Vars
+// Mixins
+@mixin button-primary {
+    background-color: $blue;
+    text-decoration: none;
+    user-select: none;
+    padding: 1rem 1.5rem;
+    border-radius: 6px;
+    font-weight: bold;
+    text-transform: uppercase;
+    text-align: center;
+    display: inline-block;
+    color: $off-white;
+    font-size: 1.3rem;
+    cursor: pointer;
+    transition: 0.2s all;
+
+    &:hover {
+        background-color: $dark-blue;
+    }
+
+    &:active {
+        transform: translateY(4px);
+        color: $off-white;
+        background-color: $black;
+    }
 
-:root {
-    --logo-height: 100px;
-    --main-margin: 30px; // flex gap not available in Safari :(
+    @media screen and (max-width: 900px) {
+        font-size: 1em;
+    }
 }
 
-// Element styles
+@mixin button-secondary {
+    @include button-primary;
+    background-color: white;
+    color: $blue;
+    border: 2px solid $blue;
 
-* {
-    box-sizing: border-box;
+    &:hover {
+        color: $off-white;
+    }
 }
 
-html, body {
-    overflow: hidden;
-    height: 100%;
-    width: 100%;
+// General Styles
+* {
     margin: 0;
     padding: 0;
+    box-sizing: border-box;
+    font-family: 'Work Sans', sans-serif;
```

Comment:
Are we hosting this font in `assets/`? Not provided on macOS by default at least

Reply:
They are imported in HTML via google fonts (see lines 31 - 34 of `index.html`)

Reply:
Oops, I missed the `Work+Sans` when searching for `Work Sans` in Sublime 🤦

Reply:
It isn't working on your system, though? It appears to be working on mine. I believe we use the same font in the club site, is it showing up there?

Reply:
Yep! works for me locally, was just commenting generally if we didnt improts thu GFonts